City-of-Helsinki/drupal-helfi
# UHF-3 Multilingual support

How to test

- `make new`
- check Drupal UI for 4 languages at `/admin/config/regional/language`
- check that lang negotiation settings are sensible at `admin/config/regional/language/detection` (only Account administration pages and URL enabled + URL langcodes per lang OK)
- add content type (if none exist) and save a new item with any language - you should be redirected to corresponding language on each submit (check URL)
